lib/backend/backend.go

```diff
@@ -410,7 +410,7 @@ func Key(parts ...string) []byte {
 
 // ExactKey is like Key, except a Separator is appended to the result
 // path of Key. This is to ensure range matching of a path will only
-// math child paths and not other paths that have the resulting path
+// match child paths and not other paths that have the resulting path
```

Comment:
this is pretty unrelated to the RFD, I'd leave it out of the PR

Reply:
https://github.com/gravitational/teleport/pull/15706/commits/2c954dd58f54a58d8061d97168cccbc6bf57b98c